recipes-core/uclient/uclient/0001-Disable-warnings-being-treated-as-errors.patch

```diff
@@ -0,0 +1,27 @@
+From 02c692064950ea480ff9d34337b2ad1da36ed545 Mon Sep 17 00:00:00 2001
+From: Fabio Berton <fabio.berton@ossystems.com.br>
+Date: Mon, 2 May 2016 11:26:53 -0300
+Subject: [PATCH] Disable warnings being treated as errors
+Organization: O.S. Systems Software LTDA.
+
+Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
+---
+ CMakeLists.txt | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ee7d91d..2975229 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.6)
+ INCLUDE(CheckIncludeFiles)
+ 
+ PROJECT(uclient C)
+-ADD_DEFINITIONS(-Os -Wall -Werror --std=gnu99 -g3 -Wmissing-declarations)
++ADD_DEFINITIONS(-Os -Wall --std=gnu99 -g3 -Wmissing-declarations)
```

Comment:
Just tested building without this commit, and am not seeing the build failure that Fabio saw. I will drop this commit from my pending new pull request.